solidity/contracts/libs/Call.sol

```diff
@@ -9,43 +9,78 @@ library CallLib {
         bytes data;
     }
 
-    function _multicall(address to, bytes[] memory calls) internal {
+    struct CallWithValue {
+        uint256 value;
+        Call _call;
+    }
+
+    struct CallWithCallback {
+        Call _call;
+        bytes callbackdata;
+    }
+
+    function _call(Call memory call) internal returns (bytes memory) {
```

Comment:
Should the functions prefixed with `_` be private? If not, why use it in call but not multicall?